TestsProject/TestsProjectTests/InitializationTests.m

```diff
@@ -131,95 +131,91 @@ - (void*)pixForImage:(UIImage *)image;
         });
     });
 
-    NSString *customTessDataPath = [customDirectoryPath stringByAppendingPathComponent:@"tessdata"];
-    void(^cleanCustomTessdataFolder)() = ^{
+    NSString *tessdataPath = @"foo/bar";
+    void (^cleanTessdataFolderAtPath)(NSString *dataPath) = ^(NSString *dataPath) {
+        //NSLog(@"Removing previous tessdata folder from Caches folder");
         NSError *error = nil;
-        BOOL fileIsRemoved = [fileManager removeItemAtPath:customTessDataPath error:&error];
+        BOOL fileIsRemoved = [fileManager removeItemAtPath:dataPath error:&error];
         if (error != nil) {
-            NSLog(@"Error deleting tessdata folder from the custom directory: %@", error);
+            NSLog(@"Error deleting tessdata folder from the Caches folder: %@", error);
```

Comment:
You refer to the path in this helper function as the "Caches folder" and its contents as `cachesContent`, but isn't it really just whatever path is passed in as `dataPath`? I think this should be changed because the Caches folder has a specific meaning in iOS and I don't think this function is using that meaning anymore.